picky-krb/src/crypto/aes/key_derivation.rs

```diff
@@ -13,7 +13,7 @@ use super::AesSize;
 /// default string-to-key parameters        00 00 10 00
 const AES_ITERATION_COUNT: u32 = 0x1000;
 
-fn random_to_key(data: Vec<u8>) -> Vec<u8> {
+pub fn random_to_key(data: Vec<u8>) -> Vec<u8> {
     data
 }
```

Comment:
The input is returned as-is. Is that intended?

Reply:
Yes, it is intended. I made this function more for code readability than for functionality. According to RFC 3961, we have the rundom_to_key function. In AES encryption it does nothing(`irandom-to-key function: identity function` https://www.rfc-editor.org/rfc/rfc3962.html#section-6)

Reply:
Makes sense! I think it would help to have this as a comment in the code 🙂

Reply:
I added a comment